### ISSUE TITLE
Documentation bug about grid

### ISSUE THREAD

**Issue by Rurik19** [Dogfalo/materialize#3887]
There is bug in materializecss documentation. on http://materializecss.com/grid.html it sais 

> Note: For now, just know that the s1 stands for small-1 which in plain English means "1 column on small screens".

Really the matter is upside-down: it is **s12** for one column in small screen :)

`.row .col.s12 {
  width: 100%;
  margin-left: auto;
  left: auto;
  right: auto;
}`

**Comment by hulkinBrain** [comment]
It means 1 column wide width assuming we are using the grid layout which has 12 columns. It's not a bug, its just a matter of contextual understanding.
